packages/platforms/browser/lib/auto-instrumentation/resource-load-plugin.ts

```diff
@@ -7,6 +7,8 @@ interface ResourceTiming extends PerformanceResourceTiming {
 
 export function getHttpVersion (protocol: string) {
   switch (protocol) {
```

Comment:
Would changing the `default` instead make sense?

Reply:
probably! I guess it depends if we want to exclude all of the other [possibilities of nextHopProtocol](https://www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml#alpn-protocol-ids) ?

@gareththackeray any thoughts on this?

Reply:
The product design lists the values for `http.flavor` as `1.0 | 1.1 | 2.0 | 3.0 | SPDY | QUIC` so have gone ahead and updated it to return `undefined` for anything outside of this range.

Reply:
I think use watever you get back if it has a value

Reply:
so, just to clarify - we want to map the values to those as described in the PD (such as `h2` and h2c` to `2.0`) and anything outside of that range, we use as it is? like `xmpp` or `stun.turn`

Reply:
Yes I think so